src/content/docs/en/pages/marketplace/marketplace.mdx

```diff
@@ -25,98 +28,77 @@ Azion Marketplace offers several advantages:
+Check [Azion’s guides page](/en/documentation/products/guides/#marketplace) for more details on the installation for type of integration.
 
-To make sure that you’re running the latest version of the solution, go to Azion Marketplace and select the solution you want to update. By clicking the **Get new version button**, you'll download the solution’s latest version.
+<Button href="/en/documentation/products/marketplace/integrations/" text="go to integrations reference" variant="secondary"> 
 
-A message informing you the solution has been successfully updated will appear. Afterward, you can view all your solutions in your [Edge Functions list](https://manager.azion.com/edge-functions).
+</Button>
+<br />
+<Button href="/en/documentation/products/marketplace/install-an-integration/" text="go to install an integration guide" variant="secondary"> 
 
-> Important: when you update a solution in Azion Marketplace, it won’t overwrite the version of the solution’s version you’re currently running on your edge application. This occurs to give you control over the version you want to use with your application.
->
-> For instance, you could test a new version of a solution with a different edge application. So, when the new version has passed through your validation process, on your edge application, you can select it to run. To do so, change the solution version on the functions dropdown menu of your edge application.
+</Button>
 
 ---
 
 ## How-to guides
 
-Each solution may have different configurations and dependencies to be installed and for them to work adequately. To find out how to configure a specific solution and be able to install it successfully, go to the [Azion's guides page](/en/documentation/products/guides/#marketplace) and select the guide related to the solution you want to use.
+Each solution may have different configurations and dependencies to be installed and for them to work adequately. To find out how to configure a specific solution and be able to install it successfully, go to the [Azion's guides page](/en/documentation/products/guides/) and select the guide related to the solution you want to use.
 
 ---
 
-## Permissions
+## Solutions' permissions
 
-In order to use Marketplace properly, some solutions will need permissions to execute repeated actions on your account. This happens because some solutions have semi-auto installations. To understand more about the permissions, you can visit the [Marketplace Permissions](/en/documentation/products/guides/permissions-mktp/) documentation page. 
+In order to use Marketplace properly, some solutions will need permissions or comply to certain requirements. This happens because some solutions have semi-auto installations or work with thirs-party integrations. To understand more about the permissions, you can visit the [Marketplace Permissions](/en/documentation/products/guides/permissions-mktp/) documentation page. 
```

Comment:
```suggestion
In order to use Marketplace properly, some solutions will need permissions or comply to certain requirements. This happens because some solutions have semi-auto installations or work with third-party integrations. To understand more about the permissions, you can visit the [Marketplace Permissions](/en/documentation/products/guides/permissions-mktp/) documentation page. 
```